CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+Important changes to data-models, configuration and migrations between each
+version, listed here to easy deployment and troubleshooting.
+
+## Next Release
+
+ * Run `gcloud app deploy cron.yaml` to update cron-job retry logic.
+
+## 20190306t115839
+ 
```

Comment:
You could add runtime-version of the deployment here.

Reply:
++